portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,72 @@
+body {
+    font-family: 'Sora', Sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+
+.title {
+    font-family: 'Sora', Sans-serif;
+    font-weight: 700;
+    font-size: 48px;
+}
+
+.sub-title {
+    font-family: 'Sora', Sans-serif;
+    font-weight: 500;
+    font-size: 28px;
+}
+
+
+#home {
+  margin-left: auto;
+  margin-right: auto;
+  font-family: 'Sora', Sans-serif;
+  background-image: url('/images/home-background.jpg');
+  width: 900px;
+  background-size:900px auto;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 900px;
+  font-family: 'Sora', Sans-serif;
+  background-image: url('/images/background.jpg');
+  background-size: 900px auto;
 }
 
 #fact-container {
   margin-top: 20px;
 }
+
+.middle {
+    vertical-align:middle;
+}
+
+.work-container{
+    height:300px;
+    padding-right: 32px;
+}
+
+.sub-title {
+     font-family: 'Be Vietnam', sans-serif;
+    font-weight: 800;
+    font-size: 30px;
+}
```

Comment:
Either the one at line 13 or this one is actually in use (probably the latter). Please consider removing the one you don't use :)